<?xml version="1.0"?>
<launch>
  <arg name="robot" default="iiwa" />

  <node ns="$(arg robot)" name="path_mapper" pkg="rll_planning_project" type="path_mapper.py" respawn="false" output="screen">
  	<param name="map_width" value="1.2" />
  	<param name="map_length" value="1.6" />
  </node>

</launch>
